src/subscribers/abstract_subscriber.jl

```diff
@@ -65,17 +65,35 @@ struct SubscribedMessage
     sub::Subscriber          # Note this is an abstract type
     lock::ReentrantLock
     name::String
+    task::Vector{Task}
```

Comment:
So now the `SubscribedMessage` keeps track of its own task, is this the same task as that is tracked in `NodeIO`?

Reply:
Yep, I haven't deleted the one in `NodeIO` yet. I'll go ahead and delete it out now.

Reply:
Cleaned up. I removed the task list in `NodeIO`.